docs/mathematics/sieve-of-eratosthenes.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Sieve of Eratosthenes
+slug: /mathematics/sieve-of-eratosthenes
+---
+
+## Introduction
+
+The sieve of Eratosthenes is an algorithm for finding all prime numbers up to any given limit. It is computationally highly efficient algorithm.
+
+## Explanation
+
+Let us take the number `n = 100` , so we create list from `2` to `100` . According to the algorithm , we mark all the numbers which are divisible by `2` which are greater than or equal to the square of it . Then we move to the next number `3` , we mark all the numbers which are divisible by `3`  which are greater than or equal to the square of it. we continue this process until the square of the number is greater than `n`. The numbers which are unmarked in the list are the prime numbers less than `n`.
+
```

Comment:
If would be great if you could add some pictures to explain the workflow of the algorithm.